CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -1351,7 +1556,9 @@ CFTimeZoneRef CFTimeZoneCreateWithName(CFAllocatorRef allocator, CFStringRef nam
 	CFStringRef mapping = CFDictionaryGetValue(dict, name);
 	if (mapping) {
 	    name = mapping;
-	} else if (CFStringHasPrefix(name, __tzZoneInfo)) {
+	}
+#if !TARGET_OS_ANDROID
```

Comment:
Similar, folding this into the `if` condition would be nicer.

Reply:
Same explanation as above.